src/command/status.ts

```diff
@@ -12,17 +12,30 @@ export class Status extends RootCommand implements LeafCommand {
   public async run(): Promise<void> {
     super.init()
 
+    this.console.divider()
+    this.console.log(bold('Bee Status'))
+    this.console.divider()
     await this.checkBeeApiConnection()
     const version = await this.checkBeeDebugApiConnection()
     await this.checkBeeVersionCompatibility()
+    const topology = await this.checkTopology()
 
-    this.console.divider()
     this.console.log(createKeyValue('Bee Version', version, 'Supported Version'.length))
     this.console.log(
       createKeyValue('Supported Version', SUPPORTED_BEE_VERSION + ' (' + SUPPORTED_BEE_VERSION_EXACT + ')'),
     )
     this.console.quiet('Bee version - ' + version)
     this.console.quiet('Supported version - ' + SUPPORTED_BEE_VERSION_EXACT)
+
+    if (topology) {
+      this.console.divider()
```

Comment:
Here, I would use different devider as this is a different section
```suggestion
      this.console.divider('=')
```